main.py

```diff
@@ -335,14 +284,12 @@ def run(self):
                 # when we update the Siggy data.
 
                 self._update_npc_kills()
-                self._update_siggy_data()
+                self._update_route_data()
 
                 routes = self._find_high_delta_routes()
 
                 if len(routes) > 0:
-                    slack_msg = self._format_slack_message(routes)
-                    requests.post(SLACK_WEBHOOK, data=json.dumps(slack_msg))
-
+                    requests.post(WEBHOOK_URL, data=self._format_message(routes[:20]))
```

Comment:
Would prefer to avoid magic numbers. Could you please add a variable to the top of the file, something like `MAX_ROUTES`?